README.md

```diff
@@ -109,24 +108,25 @@ There are already the following [debug configurations](https://code.visualstudio
 -   `Launch Frontend [Firefox]` (You have to install an extra extension)
 -   `Debug Jest Tests`
 
-In addition you can make use of the following browser extensions:
+In addition, you can make use of the following browser extensions:
 
 -   [Angular DevTools](https://chrome.google.com/webstore/detail/angular-devtools/ienfalfjdbdpebioblfackkekamfmbnh)
--   [Redux DevTools Extension](https://github.com/zalmoxisus/redux-devtools-extension/) for [NgRx](https://ngrx.io/guide/store-devtools)
```

Comment:
This would need additional code in the frontend to register it. I also never needed it.